libs/blog/articles/feature-list/src/lib/category-section-container/category-section-container.component.html

```diff
@@ -2,8 +2,8 @@
   *transloco="let t"
   [title]="t(displayName())"
   [link]="{
```

Comment:
isn't that component used several times? not sure about this static string

Reply:
Yes, that's right. But we decided to change it in each section